extensions/vscode/src/utils/errorTypes.test.ts

```diff
@@ -337,5 +358,15 @@ describe("resolveAgentJsonErrorMsg", () => {
     );
 
     expect(msg).toBe(`The Configuration has a schema error on line 7`);
+
+    msg = resolveAgentJsonErrorMsg(
+      mkAxiosJsonErr({
+        code: "pythonExecNotFound",
+      }) as axiosErrorWithJson,
+    );
+
+    expect(msg).toBe(
+      "Could not find a Python executable in the system to inspect deployment requirements.",
```

Comment:
I'd argue since this same error is used anytime we encounter this:
- open a Python file and attempt to do the inspection for the entrypoint button
- select a file to create a deployment

We should simply even further and say:
"Could not find a Python executable"

We could potentially add more, but a few of these cases don't have much to do with requirements at all.

Reply:
I'm fine for even shorter — I thought giving someone a reason why might connect them to their project (as opposed to say, needing python to do something unrelated to their project). But will admit that "to inspect requirements" is probably more vague about that than it is helpful.

Reply:
Thanks! makes sense, I'll change it for the shorter message